<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_bohren</class>
 <widget class="QDialog" name="Dialog_bohren">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bohren</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_allg">
    <attribute name="title">
     <string>allgemein</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="label_x_var">
        <property name="text">
         <string>[*X]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_y_var">
        <property name="text">
         <string>[*Y]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_x"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_dm">
        <property name="text">
         <string>Durchmesser</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_wkz"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_wkz">
        <property name="text">
         <string>Werkzeug</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_wkz_var">
        <property name="text">
         <string>[WKZ]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_y">
        <property name="text">
         <string>Mittelpunkt Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_x">
        <property name="text">
         <string>Mittelpunkt X</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_y"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_boti">
        <property name="text">
         <string>Tiefe</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_boti_var">
        <property name="text">
         <string>[*BT]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_boti"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_dm">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_dm_var">
        <property name="text">
         <string>[*DM]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tech">
    <attribute name="title">
     <string>Technelogie</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_anboti_var">
        <property name="text">
         <string>[ANBT]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_zust_var">
        <property name="text">
         <string>[ZUST]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_zust"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_reboma"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_bovors">
        <property name="text">
         <string>Bohrvorschub</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_reboma_var">
        <property name="text">
         <string>[REBT]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_anboti">
        <property name="text">
         <string>Anbohrtiefe</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_kom_var">
        <property name="text">
         <string>[KOM]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_anbovors"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_bovors"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_anbovors_var">
        <property name="text">
         <string>[FAN]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_anbovors">
        <property name="text">
         <string>Anbohrvorschub</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_bez">
        <property name="text">
         <string>Bezeichnung</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lineEdit_bez"/>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lineEdit_afb"/>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_afb_var">
        <property name="text">
         <string>[AFB]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_anboti"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_afb">
        <property name="text">
         <string>Ausführbedingung</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_kom">
        <property name="text">
         <string>Kommentar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_bovors_var">
        <property name="text">
         <string>[F]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_reboma">
        <property name="text">
         <string>Restbohrmaß</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_zust">
        <property name="text">
         <string>Zustellmaß</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_kom"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_bez_var">
        <property name="text">
         <string>[BEZ]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_drehzahl">
        <property name="text">
         <string>Drehzahl</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_drehzahl_var">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_drehzahl"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_wkz">
    <attribute name="title">
     <string>Werkzeug</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="5" column="1">
       <widget class="QLabel" name="label_wkz_kom_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_autowerte">
        <property name="text">
         <string>AUTO-Werte:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_wkz_fan">
        <property name="text">
         <string>Eintauchvorschub</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_wkz_fan_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_wkz_n_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_wkz_drehri_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_wkz_nummer_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_wkz_nutzl_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_wkz_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_wkz_nummer">
        <property name="text">
         <string>Nummer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_wkz_dm_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_wkz_dm">
        <property name="text">
         <string>Durchmesser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_wkz_name_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_wkz_drehri">
        <property name="text">
         <string>Drehrichtung</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_wkz_kom">
        <property name="text">
         <string>Kommentar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_wkz_nutzl">
        <property name="text">
         <string>Nutzlänge</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_wkz_f">
        <property name="text">
         <string>Vorschub</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_wkz_n">
        <property name="text">
         <string>Drehzahl</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_wkz_f_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_speichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_wkz</tabstop>
  <tabstop>lineEdit_dm</tabstop>
  <tabstop>lineEdit_x</tabstop>
  <tabstop>lineEdit_y</tabstop>
  <tabstop>lineEdit_boti</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
  <tabstop>pushButton_speichern</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>lineEdit_anboti</tabstop>
  <tabstop>lineEdit_zust</tabstop>
  <tabstop>lineEdit_reboma</tabstop>
  <tabstop>lineEdit_anbovors</tabstop>
  <tabstop>lineEdit_bovors</tabstop>
  <tabstop>lineEdit_drehzahl</tabstop>
  <tabstop>lineEdit_kom</tabstop>
  <tabstop>lineEdit_bez</tabstop>
  <tabstop>lineEdit_afb</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
